### PR TITLE
Added files where execute key escape to exit the program.

### PR DESCRIPTION
**Pull request by ntamurejocolorado**
I have implemented an abort function where you can push escape key then the program is correctly closed; and when you push key n you can change between frames.

### CODE REVIEW

File: modules/nci/src/imanager.cpp [hunk 1]
```diff
@@ -145,4 +139,24 @@ bool IManager::run()
     return true;
 }
 
+void IManager::execute_key_if_exist(int character)
+{
+    if( KEYS::ESCAPE == character)
```
Comment (IslameN) [inline modules/nci/src/imanager.cpp:144]:
```suggestion
    if (KEYS::ESCAPE == character)
```

File: modules/nci/src/frame.cpp [hunk 2]
```diff
@@ -118,7 +118,7 @@ void Frame::run()
     if(!_runnable)
         return;
     
-    set_selected(false);
+    set_selected(true);
     draw();
     for(auto child: _children)
```
Comment (IslameN) [inline modules/nci/src/frame.cpp:123]:
```suggestion
    for (auto child: _children)
```

File: modules/nci/include/imanager.h [hunk 3]
```diff
@@ -13,6 +13,7 @@
 #include "frameContainer.h"
 #include "popup.h"
 #include "imanager.h"
+#include "keybindingContainer.h"
```
Comment (IslameN) [inline modules/nci/include/imanager.h:16]: Is it used?

**Review by IslameN — COMMENTED** [review]
It is already merged, hahahahahaha

File: modules/nci/include/imanager.h (hunk 3 above)
Comment (vgonisanz) [reply modules/nci/include/imanager.h:16]: No, this is my error, I asked for add a keybinding and finally we only quit the app when ESC is pressed.

**Pull request merged**